src/runtime/methodbinder.cs

```diff
@@ -149,6 +208,19 @@ internal static MethodInfo MatchSignatureAndParameters(MethodInfo[] mi, Type[] g
             return null;
         }
 
+        private static bool IsNullableOf(Type sigType, Type target)
+        {
+            if (!sigType.IsValueType || !target.IsValueType)
+            {
+                return false;
+            }
+            if (target != typeof(Nullable<>))
+            {
+                return false;
+            }
+            return true;
```

Comment:
Can you clarify what this method does?
It looks like for it to return `true` `target` must be `Nullable<>`, and `Nullable<int>` won't work.